cmd/mdatagen/internal/metadata/generated_metrics_test.go

```diff
@@ -1,5 +1,4 @@
 // Code generated by mdatagen. DO NOT EDIT.
-
```

Comment:
Sorry for nitpicking. This removed line brings inconsistency with generated_metrics.go. Please remove it, and we can merge.